CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # SAS Viya Monitoring for Kubernetes
 
+## UNRELEASED
+
+* **Overall**
+
+* **Metrics**
+  * [DEPRECATION] NodePorts is now disabled by default for Prometheus and AlertManager for security reasons. Set the environment variable PROM_NODEPORT_ENABLE=true to replicate previous behavior. 
```

Comment:
Would prefer describing the change more explicitly; something like: "For security reasons, access to Prometheus and AlertManager via NodePort is no longer enabled by default.  Set the environment variable PROM_NODEPORT_ENABLE=true to replicate previous behavior."

Reply:
Makes sense, will do!